adapters/mini-swe-agent/src/nemo_fabric_adapters/mini_swe_agent/adapter.py

```
@@ -37,73 +42,45 @@ class MiniSweAgentRuntime:
+        )
+        self._relay_enabled = bool(
+            runtime_context.telemetry and runtime_context.telemetry.relay_enabled
+        )
+        agent_type = RetainingDefaultAgent
+        if self._relay_enabled:
+            if importlib.util.find_spec("nemo_relay") is None:
+                raise RuntimeError(
+                    "telemetry is enabled but a compatible 'nemo-relay' package is "
+                    "not installed; install the Relay extra (pip install "
+                    "'nemo-fabric-adapters-mini-swe-agent[relay]')."
                 )
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)